bower_components/bourbon/app/assets/stylesheets/_bourbon-deprecated-upcoming.scss

```diff
@@ -0,0 +1,411 @@
+// The following features have been deprecated and will be removed in the next MAJOR version release
+
+@mixin inline-block {
+  display: inline-block;
+
+  @warn "The inline-block mixin is deprecated and will be removed in the next major version release";
+}
+
+@mixin button ($style: simple, $base-color: #4294f0, $text-size: inherit, $padding: 7px 18px) {
```

Comment:
Color literals like `#4294f0` should only be used in variable declarations; they should be referred to via variable everywhere else.